app/controllers/admin/api/settings_controller.rb

```diff
@@ -40,7 +40,13 @@ def settings_params
     @settings_params ||= params.require(:settings).permit(*ALLOWED_PARAMS)
   end
 
-  def enforce_sso_allowed?
+  def validate_enforcing_sso_allowed
+    return unless settings_params.key?(:enforce_sso)
+
+    new_value = Settings.type_for_attribute('enforce_sso').cast(settings_params[:enforce_sso])
```

Comment:
why do we need this complicated expression?

Reply:
I think we had some boolean conversion somewhere else. Would be nice to standardize on our own sane logic, e.g. accept `true`, `false`, `0`, `1`, case insensitive but I'm fine to leave it as is.